index.js

```diff
@@ -57,6 +57,20 @@ function getWebsocketConnection(
   ws.on('error', err => {
     logger.error(`Websocket error: ${err.message}`);
   });
+  ws.on('unexpected-response', (_, res) => {
```

Comment:
when does this happen? and what did you do to trigger this?

Reply:
This happens when there's an unexpected response from the market maker (usually an error code like 401).  I added this because we previously had no way of telling why the server rejected our connection request (just "401 unauthorized"). 

This way, we now see "Unauthorized IP", "Unknown MM", etc.

Reply:
To trigger, I simply tried connecting to 'AlfieMM' without having my IP allowed.